packages/firebase_auth/README.md

```diff
@@ -25,13 +25,13 @@ dependencies {
   // Example existing classpath
   classpath 'com.android.tools.build:gradle:3.0.1'
   // Add the google services classpath
-  classpath 'com.google.gms:google-services:3.1.2'
+  classpath 'com.google.gms:google-services:3.2.0'
```

Comment:
We probably want to update this across all firebase-xxx plugins, google_sign_in, and cloud_firestore. The `example/` projects should be updated as well.

To publish the change, it would need to be reflected in `CHANGELOG.md` and `pubspec.yaml` for each plugin.

Reply:
We may also want to update the compile dependencies to `com.google.firebase:firebase-xxx:12.0.1` across the board.

cc @collinjackson

Reply:
I'm assuming you meant to go after this internally, but I won't have time at this point to go through all the changes. I'd like to, but that means less prep time for recording. :)

Reply:
This is done in #500